GOVERNANCE.md

```diff
@@ -1,172 +1,74 @@
-<!-- see https://github.com/yzhang-gh/vscode-markdown/blob/master/README.md#table-of-contents -->
-<!-- omit in toc -->
-# GitOps Working Group Governance
+# Governance
 
-This document <https://github.com/gitops-working-group/gitops-working-group/blob/main/GOVERNANCE.md> defines the governance process for the OpenGitOps project and the GitOps Working Group and community.
+This document defines the governance process for the GitOps Working Group and community.
 
-- [Values](#values)
-  - [Code of Conduct](#code-of-conduct)
-- [Roles in the GitOps Working Group Community](#roles-in-the-gitops-working-group-community)
-  - [Users](#users)
-  - [Contributors](#contributors)
-  - [Maintainers](#maintainers)
-  - [Oversight Committee](#oversight-committee)
-- [Decision Making](#decision-making)
-  - [Deciders](#deciders)
-  - [Decision Guidelines](#decision-guidelines)
-  - [Simple Majority Decisions](#simple-majority-decisions)
-  - [Supermajority Decisions](#supermajority-decisions)
-  - [Unanimity Decisions](#unanimity-decisions)
-- [Proposal Process](#proposal-process)
-- [Licenses and Copyright](#licenses-and-copyright)
+## Membership
 
-## Values
+While anyone may join and contribute, voting members must be active members of the CNCF.
+Apart from following the [Code of Conduct](CODE_OF_CONDUCT.md), there are no special requirements.
+This is an open public working group welcoming anyone who would like to help.
+Feel free to register your interest in the [interested parties](interested-parties.md) document.
```

Comment:
@todaywasawesome I think we should specify the voting requirements in only one place so I linked below instead. Then the "membership" wording needed to be switched around to still make sense. WDTY?

```suggestion
This is an open public working group welcoming anyone who would like to join and help.
Apart from following the [Code of Conduct](CODE_OF_CONDUCT.md), there are no special requirements for membership.
Feel free to register your interest in the [interested parties](interested-parties.md) document.
Note that voting requires additional conditions (see [Elections](#elections)).
```